### PR TITLE
Added budget card

### PR DESCRIPTION
**Pull request by ewelina-slepko**

### CODE REVIEW

File: src/app/home/layouts/budgets-form/budgets-form.component.scss [hunk 1]
```diff
@@ -28,6 +28,112 @@
...
+.x-icon-container {
+  width: 100%;
+  display: flex;
+  justify-content: flex-end;
+}
+
+.x-icon {
+  font-size: 1.6rem;
+  margin: 1.2rem 1.6rem 0 0;
+  color: $medium-gray;
```
Comment (Rementh) [inline src/app/home/layouts/budgets-form/budgets-form.component.scss]: I found this quite nice. Gut jop.

File: src/app/home/layouts/budgets-form/budgets-form.component.scss [hunk 2]
```diff
@@ -28,6 +28,112 @@
...
+.cycle-container {
+  display: flex;
+  justify-content: center;
+}
+
+.cycle-header {
+  font-size: 0.9rem;
+  align-self: flex-start;
+  color: $medium-gray;
+  margin: 2rem 2.6rem 1rem;
```
Comment (Rementh) [inline src/app/home/layouts/budgets-form/budgets-form.component.scss]: Oh, thats good!

File: src/app/home/layouts/budgets-form/budgets-form.component.html [hunk 3]
```diff
@@ -3,7 +3,58 @@ <h2 class="header">Budgets</h2>
...
+</div>
+
+<div *ngIf="isNewBudgetCardVisible" class="budget-form-container">
+
+  <div class="x-icon-container">
+    <i class="fas fa-times x-icon" (click)="isNewBudgetCardVisible = false"></i>
+  </div>
+
+  <fieldset class="budget-fieldset">
+    <section class="budget-input-section">
```
Comment (Rementh) [inline src/app/home/layouts/budgets-form/budgets-form.component.html]: I wonder if that can be deleted.

File: src/app/home/layouts/budgets-form/budgets-form.component.html [hunk 4]
```diff
@@ -3,7 +3,58 @@ <h2 class="header">Budgets</h2>
...
+                  ngModel
+                  type="number"
+                  [isCheckIconVisible]="false"
+                  required></text-input>
+    </section>
+  </fieldset>
+
+  <p class="cycle-header">Cycle</p>
+  <div class="cycle-container">
+    <div class="cycle-element"><span class="cycle-text">Weekly</span></div>
```
Comment (Rementh) [inline src/app/home/layouts/budgets-form/budgets-form.component.html]: Try get rid of the span element.

**Comment by ewelina-slepko** [comment]
Done!

**Pull request merged**